# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/notepadnext/hotfix-stdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- src/scintilla/include/ScintillaTypes.h.vanilla	2025-06-19 23:30:33.733987231 +0200
+++ src/scintilla/include/ScintillaTypes.h	2025-06-19 23:30:40.781283723 +0200
@@ -11,6 +11,8 @@
 #ifndef SCINTILLATYPES_H
 #define SCINTILLATYPES_H
 
+#include <stdint.h>
+
 namespace Scintilla {
 
 // Enumerations
